src/views/HomePage.tsx

```diff
@@ -69,40 +69,40 @@ function HomePage(): JSX.Element {
 
   return (
     <div
-      className="h-full w-full p-28 absolute top-0 left-0"
+      className="h-full w-full flex flex-col justify-center absolute top-0 left-0"
       style={{
         backgroundImage: `url(${BG})`,
         backgroundRepeat: 'no-repeat',
         backgroundColor: 'black',
         backgroundSize: 'cover',
       }}
     >
-      <div className="text-white font-roboto container">
-        <h1 className="text-8xl font-bold">aevisio</h1>
-        <h2 className="text-2xl">Expert Comptable.Audit.Conseil</h2>
+      <div className="text-white font-roboto flex items-center  flex-col">
+        <h1 className="sm:text-9xl text-8xl font-bold">aevisio</h1>
+        <h2 className="sm:text-2xl text-2xl">Expert Comptable.Audit.Conseil</h2>
       </div>
       <form
-        className="w-5/12 mt-10 flex flex-col text-white font-roboto text-xl"
+        className="w-12/12 sm:w-full mt-5 flex items-center flex-col text-white font-roboto text-xl sm:text-2xl"
```

Comment:
w-12/12 & sm:w-full are egal.
In desktop mode, login form take full screen centered